overrides/ios-override.json

```diff
@@ -106,131 +106,131 @@
                         "reason": "Internal exclusion to roll out experiment"
                     }
                 ],
-                "ddgFixedUserAgent": {
```

Comment:
Did we check that iOS config parsing won't have an issue with this key being renamed?

Reply:
@sammacbeth  yeah there's even tests for it here: https://github.com/duckduckgo/iOS/blob/8e959f383e9c95564aaeca5a450132ddda03ede2/DuckDuckGoTests/UserAgentTests.swift#L390